community/libev/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+patch -p1 < libev-4.25-pc.patch
+
+sed -i -e "/^include_HEADERS/s/ event.h//" Makefile.am
+
+./configure \
+    --disable-maintainer-mode \
```

Comment:
What is the default value of `maintainer-mode`? If it's "disabled" or "off" this can be removed.

Reply:
Good catch:

> dnl maintainer-mode's default is 'disable' unless 'enable' is passed

This isn't needed.